diff --git a/modules/drivers/deps.edn b/modules/drivers/deps.edn
index 98f65a210d..e0c8da9d68 100644
--- a/modules/drivers/deps.edn
+++ b/modules/drivers/deps.edn
@@ -23,4 +23,6 @@
   metabase/sqlite             {:local/root "sqlite"}
   metabase/sqlserver          {:local/root "sqlserver"}
   metabase/starburst          {:local/root "starburst"}
-  metabase/vertica            {:local/root "vertica"}}}
+  metabase/vertica            {:local/root "vertica"}
+  metabase/duckdb             {:local/root "duckdb"}
+  }}
